Sources/BrowserServicesKit/ContentBlocking/ContentBlockingRulesCompilationTask.swift

```diff
@@ -65,6 +70,7 @@ extension ContentBlockerRulesManager {
                 DispatchQueue.main.async {
                     let identifier = model.rulesIdentifier.stringValue
                     Logger.contentBlocking.debug("Lookup CBR with \(identifier, privacy: .public)")
+                    // Todo: how do we exclude this case from compilation time where the result is returned from cache
```

Comment:
You could change result to struct, and have it contain nested field e.g. `PerformanceInfo` which would also be a struct and could contain time information and other relevant metadata used to send metrics.